crypto/lhash/lhash.c

```diff
@@ -35,6 +35,10 @@
  * https://pdfs.semanticscholar.org/ff4d/1c5deca6269cc316bfd952172284dbf610ee.pdf
  */
 
+#if !defined(OPENSSL_BUILDING_OPENSSL)
+#error wtf?
```

Comment:
Please don‘t use x-rated language.

Reply:
It is intended to be a temporary commit to help diagnose why the build's are broken. But point taken, I updated it.